pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -5327,7 +5327,13 @@ def test_downsample_in_time(resample_in_time_file):
     out_ns = uv_object.get_nsamples(0, 1)
     assert np.isclose((init_ns[0, 0, 0] + init_ns[1, 0, 0]) / 2.0, out_ns[0, 0, 0])
 
-    return
+    # Compare doing it with n_times_to_avg
+    uv_object2.downsample_in_time(
+        n_times_to_avg=2, blt_order="baseline", minor_order="time"
+    )
+    assert uv_object.history != uv_object2.history
```

Comment:
Why do the histories end up being different here? (Btw, I think this is a good rigorous check for making the histories equal.)

Reply:
I see that you figured this out. Do you want me to add a comment?

Reply:
Yeah, a quick comment might be helpful to those of us who form opinions before reading code :)